run/GCHP/ExtData.rc.templates/ExtData.rc.CO2

```diff
@@ -152,7 +152,7 @@ CMSF_CO2_BN     kgC/km2/s N Y F%y4-%m2-01T00:00:00   none   1e-6 CO2_Flux /nobac
 #==============================================================================
 # --- Time zones (offset to UTC) ---
 #==============================================================================
-TIMEZONES count N V - none none UTC_OFFSET ./HcoDir/TIMEZONES/v2015-02/timezones_voronoi_1x1.nc
+TIMEZONES count Y Y 2017-%m2-01T00:00:00 none none UTC_OFFSET ./HcoDir/TIMEZONES/v2024-02/timezones_vohra_2017_0.1x0.1.nc
```

Comment:
Why is this "Y Y" and not "Y V"?

Reply:
Good catch.  I'll fix it.  Should this be "N V" to match the prior code?